pypeit/core/gui/identify.py

```diff
@@ -870,43 +870,43 @@ def store_solution(self, final_fit, binspec, rmstol=0.15,
-                        ow_wvcalib = input('Proceed with overwrite? (y/[n]): ')
-
-                if ow_wvcalib == 'y':
-                    wvcalib.to_file()
-                    if multi:
-                        slit_list_str = ''; slit_list = np.arange(np.shape(specdata)[0])
-                        for islit in slit_list:
-                            if islit < len(slit_list) - 1:
-                                slit_list_str += str(islit) + ','
-                            else: slit_list_str += str(islit)
-
-                        if slits:
-                            print('               ')
-                            msgs.info('Unflagging Slits from WaveCalib: ')
-                            slits.mask = np.zeros(slits.nslits, dtype=slits.bitmask.minimum_dtype())
-                            slits.ech_order = order_vec
-                            slits.to_file()
-                            print('               ')
-                    print('         ')
-                    # ask to clean up the Calibrations directory only if force_save=False, otherwise don't clean up
+                    # Ask if overwrite the existing WVCalib file only if force_save=False, otherwise don't overwrite
+                    ow_wvcalib = ''
                     if not force_save:
-                        clean_calib = input('Clean up the Calibrations/ directory? This will delete all of the existing'
-                                            ' calibrations except the Arcs and WaveCalib files. y/[n]: ')
-                        if clean_calib == 'y':
-                            cal_root = Path('Calibrations').resolve()
-                            for cal in ['Tilt', 'Flat', 'Edge', 'Slit']:
-                                for f in cal_root.glob(f'{cal}*'):
-                                    f.unlink()
+                        while ow_wvcalib != 'y' and ow_wvcalib != 'n':
+                            print('')
+                            msgs.warn('Do you want to overwrite existing Calibrations/WaveCalib*.fits file? ' + msgs.newline() +
+                                    'NOTE: To use this WaveCalib file the user will need to delete the other files in Calibrations/ ' + msgs.newline() +
+                                    ' and re-run run_pypeit. ')
+                            print('')
+                            ow_wvcalib = input('Proceed with overwrite? (y/[n]): ')
+
+                    if ow_wvcalib == 'y':
+                        wvcalib.to_file()
+                        if multi:
+                            slit_list_str = ''; slit_list = np.arange(np.shape(specdata)[0])
+                            for islit in slit_list:
+                                if islit < len(slit_list) - 1:
+                                    slit_list_str += str(islit) + ','
+                                else: slit_list_str += str(islit)
+
+                            if slits:
+                                print('               ')
```

Comment:
```suggestion
                                print('  '*10)
```
it is a little bit better to read :-)

Reply:
Thank you!